gcc/config/riscv/predicates.md

```diff
@@ -74,12 +74,20 @@
   if (GET_MODE_SIZE (mode) > UNITS_PER_WORD)
     return false;
 
+  /* Check whether the constant can be loaded in a single
+     instruction with zb* extensions.  */
+  if (TARGET_64BIT)
+    {
+      if (TARGET_ZBS && SINGLE_BIT_MASK_OPERAND (INTVAL (op)))
+	return false;
+
+      if (TARGET_ZBB && ZERO_EXTENDED_SMALL_OPERAND (INTVAL (op)))
```

Comment:
This is addiwu which got removed.  I think allo of the ZERO_EXTENDED_SMALL_OPERAND stuff needs to come out.

Reply:
Removed